IdentityCore/src/workplacejoin/mac/MSIDWorkPlaceJoinUtil.m

```diff
@@ -158,4 +158,50 @@ + (SecIdentityRef)copyWPJIdentity:(__unused id<MSIDRequestContext>)context
     return identityRef; //Caller must call CFRelease
 }
 
++ (nullable NSString*) getWPJStringData:(id<MSIDRequestContext>)context
```

Comment:
seems like iOS and macOS code are quite duplicate. Is there a possibility to reuse common logic?

Reply:
True, SecItemCopyMatching call and result processing is same.
So only that piece of code will go in some util or base class.

But IMO we can keep as it is because then one function will build dictionary and other will call Sec* API and process the result.

Any thoughts?

Reply:
I like your idea of a base class. This would help to avoid duplicate code.

Reply:
Done